snippets/specs.json

```diff
@@ -2,22 +2,22 @@
     "requires": {
         "prefix": "requires",
         "body": [
-            "requires=\"${1}\""
+            "requires(${1})"
```

Comment:
Would it not be more useful to insert `#[requires(...)]` (including the meta syntax)?

Reply:
The full `#[requires(${1})]` doesn't work very well, because by default the IDE automatically adds a `]` when the `[` is typed and the result becomes `#[requires(...)]]`. I could use just `#[requires(${1})` for the body, but I don't relying on the autocompletion of `]`. Anyway, I don't have a strong opinion on this.

Reply:
That is pretty strange, seeing as even `vscode-rust` uses this:

https://github.com/rust-lang/vscode-rust/blob/526556abd1fe981e9827d4b5c533e719fa671377/snippets/rust.json#L53

Reply:
I see, in `vscode-rust` typing `#[der<enter>` generates `#[#[derive(..)]]`, but probably the intended use case is `der<enter>` → `#[derive(..)]`, which is not an autocompletion. So far we used `#[req<enter>` → `#[requires(..)]`, but I guess it's more likely for users to know `vscode-rust`.